ui/admin/tests/acceptance/credential-library/credential-library-test.js

```diff
@@ -137,6 +137,112 @@ module('Acceptance | credential-library', function (hooks) {
     );
   });
 
+  test('can save changes to existing credential library', async function (assert) {
+    assert.expect(3);
+    assert.notEqual(instances.credentialLibrary.name, 'random string');
+    await visit(urls.credentialLibrary);
+    await click('form [type="button"]', 'Activate edit mode');
+    await fillIn('[name="name"]', 'random string');
+    await click('.rose-form-actions [type="submit"]');
+    assert.equal(currentURL(), urls.credentialLibrary);
+    assert.equal(
+      this.server.schema.credentialLibraries.all().models[0].name,
+      'random string'
+    );
+  });
+
+  test('can cancel changes to existing credential library', async function (assert) {
+    assert.expect(2);
+    await visit(urls.credentialLibrary);
+    await click('form [type="button"]', 'Activate edit mode');
+    await fillIn('[name="name"]', 'random string');
+    await click('.rose-form-actions [type="button"]');
+    assert.notEqual(instances.credentialStore.name, 'random string');
```

Comment:
To be confident that this is true, you may want to look up the instance at assertion time (like in the save case above).  This is because these instances may not be updated on change (they're more like "snapshots").

Reply:
Makes sense, I will do the change.

Since I copy this pattern from `host-catalogs` and also reproduce it on `credential-stores` test. I will provide on a separate PR these test following this pattern.